device/common/include/traccc/clusterization/device/impl/ccl_kernel.ipp

```diff
@@ -65,7 +65,9 @@ TRACCC_DEVICE void fast_sv_1(
              ++tst) {
             const details::index_t cid = tst * blckDim + tid;
 
+#ifndef ALPAKA_ACC_CPU_B_SEQ_T_SEQ_ENABLED
             __builtin_assume(adjc[tst] <= 8);
+#endif
```

Comment:
Somewhat lucky we haven't hit this sort of thing before....

`gcc` doesn't support this, and we are a pretty long way off the support in the C++ standard being adopted everywhere (an equivalent is part of C++23 I believe?).

For now, I've ignored it like this, but any actually sensible suggestions welcome, rather than this weird hack.

Reply:
The best solution here would be for us to define an `ASSUME` macro that either does something if the compiler, or is a noop if not.